tests/functional/testOnboarding.js

```diff
@@ -12,24 +12,23 @@ describe('Onboarding', function() {
 
   beforeEach(async () => {
     await vpn.flipFeatureOn("newOnboarding");
-    assert.equal(await vpn.getSetting('onboardingStarted'), false);
     assert.equal(await vpn.getSetting('onboardingCompleted'), false);
+    assert.equal(await vpn.getSetting('onboardingStep'), 0);
     await vpn.authenticateInApp();
   });
 
   //Tests pretty much all of the new onboarding, including all slides, controls, navigation, and ensuring settings persist into StateMain
```

Comment:
I know this is not part of this PR, but why use a single test to test so many different things? If possible, it would be ideal to make the tests more specific. It makes it easier to debug breakage and understand the test.

Reply:
Totally fine not to do this on this PR, but in that case please file a ticket to fo it as a follow up.